### ISSUE TITLE
Typos and errors in nb_ch02_01.ipynb

### ISSUE THREAD

**Issue by welshboi** [tensorchiefs/dl_book#6]
- linar --> linear
- curtosis --> kurtosis
- exactely --> exactly
- performace --> performance
- featues --> features
- # Here we use extract the two features and the label of the dataset --> # Here we extract the two features and the label of the dataset
- seperation --> separation
- seperate --> separate
- # training of the model using the training data stored in X and Y for 4100 epochs --> # training of the model using the training data stored in X and Y
- this values can vary --> these values can vary
- descriped --> described
- Definition of the network with two hidden layers --> Definition of the network with two layers  (note: there is only 1 hidden layer)
- paramters --> parameters
- alot --> a lot
- leraning --> learning
